libs/designsystem/src/lib/components/tabs/tabs.component.scss

```diff
@@ -1,9 +1,23 @@
 @import '../../scss/utils';
 
+:host {
+  &:not(.hide-tab-bar) {
+    // we are not hiding tabs if below large screen and not touch
+    @include media('<large') {
+      --kirby-page-footer-safe-area-bottom: 0px;
+    }
+    @include touch {
+      --kirby-page-footer-safe-area-bottom: 0px;
+    }
```

Comment:
Do we need the media queries at all? Shouldn't we always set page-footer safe area to 0 if tab-bar is shown?